scene/main/node.cpp

```diff
@@ -1289,6 +1289,29 @@ Node *Node::_get_child_by_name(const StringName &p_name) const {
 	}
 }
 
+int Node::get_descendant_count(bool p_include_internal) const {
+	int count = 0;
+	TypedArray<Node> children = get_children(p_include_internal);
+	count += children.size();
+	for (int i = 0; i < children.size(); i++) {
+		Node *node = Object::cast_to<Node>(children[i]);
+		if (node->get_child_count(p_include_internal) > 0) {
+			count += node->get_descendant_count(p_include_internal);
+		}
+	}
+	return count;
+}
+TypedArray<Node> Node::get_descendants(bool p_include_internal) const {
```

Comment:
For style there should be a line between these two functions, to make it easier to navigate and read